truthy-falsy.js

```diff
@@ -0,0 +1,206 @@
+// 1
+function setAlarm(isEmployed, isOnVacation){
+  return isEmployed && !isOnVacation;
+}
+console.log(setAlarm(true, false));
+
+// 2
+function closeCompare(firstNumber, secondNumber, numbersMargin = 0){
+  const numbersDiff = Math.abs(firstNumber - secondNumber);
+  if (numbersDiff <= numbersMargin) {
+    return 0;
+  }
+  if (firstNumber < secondNumber) {
+    return -1;
+  }
+  return 1;
+}
+
+// 3
+function or(firstVariable, secondVariable) {
+  if (firstVariable === true) {
+    return true;
+  }
+  if (secondVariable === true) {
+    return true;
+  }
+  return false;
+}
+console.log(or(true, false));
+
+function xor(firstVariable, secondVariable) {
+  if (firstVariable && !secondVariable) {
+    return true;
+  }
+  if (secondVariable && !firstVariable) {
+    return true;
+  }
+  return false;
+}
+console.log(xor(true, true));
+
+// 4
+function isTruthy(bool, firstFunction, secondFuntion) {
```

Comment:
Please don't use abbreviations

Reply:
There is a typo in `secondFuntion`

Reply:
The names `firstFunction` and `secondFunction` are too generic. We know more about them, for example when we want to call them